### PR TITLE
[TACHYON-1604] Add BlockWorkerTest unit test

### PR DESCRIPTION
**Pull request by luoli523**
Resume the unit test used in the BlockDataManagerTest

Will add more case later

### CODE REVIEW

**Comment by luoli523** [comment]
The build fails are all because of checkstyle check. All resolved.
Sorry for the mess

File: core/server/src/test/java/tachyon/worker/block/BlockWorkerTest.java [hunk 1]
```diff
@@ -49,84 +62,89 @@
...
+  private Random mRandom;
+  private Sessions mSessions;
+  private long mWorkerId;
+  private BlockWorker mBlockWorker;
 
   /**
    * Sets up all dependencies before a test runs.
    *
-   * @throws IOException if creating the test harness fails
+   * @throws IOException if initialize fails
```
Comment (yupeng9) [inline core/server/src/test/java/tachyon/worker/block/BlockWorkerTest.java]: initialize -> initialization

File: core/server/src/test/java/tachyon/worker/block/BlockWorkerTest.java [hunk 2]
```diff
@@ -49,84 +62,89 @@
...
+    Whitebox.setInternalState(mBlockWorker, "mFileSystemMasterClient", mFileSystemMasterClient);
+    Whitebox.setInternalState(mBlockWorker, "mBlockStore", mBlockStore);
+    Whitebox.setInternalState(mBlockWorker, "mHeartbeatReporter", mHeartbeatReporter);
+    Whitebox.setInternalState(mBlockWorker, "mMetricsReporter", mMetricsReporter);
+    Whitebox.setInternalState(mBlockWorker, "mSessions", mSessions);
   }
 
   /**
-   * Tests the {@link BlockDataManager#abortBlock(long, long)} method.
+   * Stop the DataServer to clean up
```
Comment (yupeng9) [inline core/server/src/test/java/tachyon/worker/block/BlockWorkerTest.java]: Stop -> Stops

**Comment by yupeng9** [comment]
Thanks for adding the test coverage. I left some minor comments

**Comment by luoli523** [comment]
The PinCommandTest fail seems has nothing to do with this patch.
I will see what's going on

**Comment by yupeng9** [comment]
LGTM

**Comment by apc999** [comment]
LGTM

Thanks for adding back the unit tests

**Pull request merged**